server/enrollment/enrollment.controller.js

```diff
@@ -35,6 +37,33 @@ function create({ body }, res) {
     .then(enrollment => res.jsend.success(processOutput(enrollment)));
 }
 
+function bulkEnroll({ body }, res) {
+  const { users, programId } = body;
+  const enrollMessages = [];
```

Comment:
please omit enroll messages, you dont need report for every user what happened

if you want you can just return number of already enrolled users